sim/side.ts

```diff
@@ -98,11 +99,13 @@ export class Side {
 		this.currentRequest = '';
 		this.maxTeamSize = 6;
 
-		this.id = sideNum ? 'p2' : 'p1';
+		let allyOffset = this.battle.gameType === 'multi' ? (sideNum < 3 ? 2 : -2) : 0;
+		this.id = (['p1', 'p2', 'p3', 'p4'] as SideID[])[sideNum + allyOffset];
```

Comment:
?????

```suggestion
		this.id = `p${sideNum + 1}` as SideID;
```

What's with this `allyOffset` thing?

Reply:
It's so I can lie to the client- uh, I mean it's to make it easier for the client to understand where this Pokemon is, but on second thought it would make much more sense to edit how `Pokemon#fullName` is set than to change `Side#id`.

Reply:
The client should also be updated to understand it normally. :[